model/bin/ww3_from_ftp.sh

```diff
@@ -10,19 +10,32 @@ curr_dir=`pwd`
 # Set WW3 code version
 ww3ver=v7.12.6
 
+interactive='y'
+if [ $# -eq 3 ] ; then
+  interactive='n'
+  ww3dir=$1  # [ '../../']
+  wnew=$2    # ['y']
+  wnew2=$3   # ['y']
+fi
+
 #Get top level directory of ww3 from user: 
 echo -e "\n\n This script will download data from the ftp for WAVEWATCH III "
 echo -e "Enter the relative path to the main/top level directory, this would "
 echo -e "be '../../' if in the model/bin directory or '.' if already in the "
 echo -e "top/main directory:"
-read ww3dir 
+if [ "$interactive" = "n" ]
```

Comment:
This is nice as it allows for previous functionality//backwards compatibility, but we're at a point where we're about to have some big changes and therefore are we willing to break this and just make this a script where you have to give it the model directory path and then you either keep things (optional, make an option) or the default is you do nothing and it doesn't keep things?